nsis/tesseract.nsi

```diff
@@ -0,0 +1,1376 @@
+    AddSize 3440
+    !insertmacro Download_Lang_Data kan
+  SectionEnd
+
+  Section /o "Georgian" SecLang_kat
+    AddSize 2410
+    !insertmacro Download_Lang_Data kat
+  SectionEnd
+
+  Section /o "Georgian (Old)" SecLang_kat_old
+    AddSize 413
+    !insertmacro Download_Lang_Data kat_old
+  SectionEnd
+
+  Section /o "Kazakh" SecLang_kaz
+    AddSize 4520
+    !insertmacro Download_Lang_Data kaz
+  SectionEnd
+
+  Section /o "Central Khmer" SecLang_khm
+    AddSize 1380
+    !insertmacro Download_Lang_Data khm
+  SectionEnd
+
+  Section /o "Kirghiz" SecLang_kir
+    AddSize 9470
+    !insertmacro Download_Lang_Data kir
+  SectionEnd
+
+  Section /o "Korean" SecLang_kor
+    AddSize 1600
+    !insertmacro Download_Lang_Data kor
+  SectionEnd
+
+  Section /o "Kurdish (Arabic)" SecLang_kur-ara
+    AddSize 3400
+    !insertmacro Download_Lang_Data kur-ara
+  SectionEnd
+
+  Section /o "Lao" SecLang_lao
+    AddSize 6090
+    !insertmacro Download_Lang_Data lao
+  SectionEnd
+
+  Section /o "Latin" SecLang_lat
+    AddSize 3040
+    !insertmacro Download_Lang_Data lat
+  SectionEnd
+
+  Section /o "Latvian" SecLang_lav
+    AddSize 2590
+    !insertmacro Download_Lang_Data lav
+  SectionEnd
+
+  Section /o "Lithuanian" SecLang_lit
+    AddSize 3010
+    !insertmacro Download_Lang_Data lit
+  SectionEnd
+  
+  Section /o "Luxembourgish" SecLang_ltz
+    AddSize 2490
+    !insertmacro Download_Lang_Data ltz
+  SectionEnd
+  
+  Section /o "Malayalam" SecLang_mal
+    AddSize 5030
+    !insertmacro Download_Lang_Data mal
+  SectionEnd
+  
+  Section /o "Marathi" SecLang_mar
+    AddSize 2020
+    !insertmacro Download_Lang_Data mar
+  SectionEnd
+
+  Section /o "Macedonian" SecLang_mkd
+    AddSize 1530
+    !insertmacro Download_Lang_Data mkd
+  SectionEnd
+
+  Section /o "Maltese" SecLang_mlt
+    AddSize 2200
+    !insertmacro Download_Lang_Data mlt
+  SectionEnd
+  
+  Section /o "Mongolian" SecLang_mon
+    AddSize 2040
+    !insertmacro Download_Lang_Data mon
+  SectionEnd
+
+  Section /o "Maori" SecLang_mri
+    AddSize 843
+    !insertmacro Download_Lang_Data mri
+  SectionEnd
+  
+  Section /o "Malay" SecLang_msa
+    AddSize 1670
+    !insertmacro Download_Lang_Data msa
+  SectionEnd
+
+  Section /o "Burmese" SecLang_mya
+    AddSize 4430
+    !insertmacro Download_Lang_Data mya
+  SectionEnd
+
+  Section /o "Nepali" SecLang_nep
+    AddSize 979
+    !insertmacro Download_Lang_Data nep
+  SectionEnd
+
+  Section /o "Dutch; Flemish" SecLang_nld
+    AddSize 5770
+    !insertmacro Download_Lang_Data nld
+  SectionEnd 
+  
+  Section /o "Norwegian" SecLang_nor
+    AddSize 3440
+    !insertmacro Download_Lang_Data nor
+  SectionEnd
+
+  Section /o "Occitan (post 1500)" SecLang_oci
+    AddSize 6030
+    !insertmacro Download_Lang_Data oci
+  SectionEnd  
+  
+  Section /o "Oriya" SecLang_ori
+    AddSize 1410
+    !insertmacro Download_Lang_Data ori
+  SectionEnd
+
+  Section /o "script and orientation" SecLang_osd
```

Comment:
This section must be removed because `osd.traineddata` is already part of the default installation (it is always needed).